minimesos/src/main/java/com/containersol/minimesos/main/CommandPs.java

```diff
@@ -0,0 +1,60 @@
+package com.containersol.minimesos.main;
+
+import com.containersol.minimesos.MinimesosException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.containersol.minimesos.MinimesosException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)